_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -213,8 +213,30 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  resetNullValues(objectData) {
+    const context = this;
+    _.forIn(objectData, (value, key) => {
+      if (value === null) {
+        _.set(objectData, key, '');
+      } else if (typeof (value) === 'object') {
+        context.resetNullValues(value);
+      }
+    });
+
+    return objectData;
```

Comment:
I'm having some trouble getting `_.mapValues` to work. It seems to work fine until the function is called again recursively, and I think that's messing up the return value. Still debugging...